### DISCUSSION TITLE
Add `createScalarFunction` into `AsyncDuckDB`/`AsyncDuckDBConnection`

### DISCUSSION THREAD

**Discussion by kolmakova** [duckdb/duckdb-wasm#1710] [Q&A]
DuckDB-WASM has a function called [createScalarFunction](https://shell.duckdb.org/docs/classes/index.DuckDBConnection.html#createScalarFunction) in `DuckDBConnection` to create used-defined functions. However, in the current version (1.28.1-dev179.0) I haven't been able to find a way to use it because `AsyncDuckDB` and `AsyncDuckDBConnection` don't implement this method. It seems to have been just forgotten to implement. Or are there some reasons why we can't use it in `AsyncDuckDB` and `AsyncDuckDBConnection`?

**Comment by rossbailey-trace** [comment]
This would be great to have!
